lib/views/pages/auth_page.dart

```diff
@@ -1,12 +1,29 @@
 import 'package:cycle_lock/controllers/auth_controller.dart';
 import 'package:cycle_lock/views/pages/screens/carousel_screen.dart';
-import 'package:cycle_lock/views/widgets/dauth_button.dart';
+import 'package:cycle_lock/views/widgets/dauth_button.dart' as summa;
 import 'package:cycle_lock/views/widgets/carousel_dots.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_dauth/flutter_dauth.dart';
 import 'package:get/get_state_manager/get_state_manager.dart';
+import 'package:flutter_dauth/flutter_dauth.dart' as dauth;
 
+// ignore: must_be_immutable
 class AuthPage extends GetView<AuthController> {
-  const AuthPage({Key? key}) : super(key: key);
+  AuthPage({Key? key}) : super(key: key);
+
+  final String _exampleText = '';
+
+  final dauth.TokenRequest _request = TokenRequest(
+      //Your Client-Id provided by Dauth Server at the time of registration.
+      clientId: '4W9RtBUEp3_9sF80',
+      //redirectUri provided by You to Dauth Server at the time of registration.
+      redirectUri: 'https://delta.nitt.edu/',
+      //A String which will retured with access_token for token verification in client side.
+      state: 'STATE',
+      //setting isUser to true to retrive UserDetails in ResourceResponse from Dauth server.
+      scope: const dauth.Scope(isUser: true),
+      //codeChallengeMethod Should be specified as `plain` or `S256` based on thier requirement.
+      codeChallengeMethod: 'S256');
```

Comment:
Would be better if this files are in a file which is not pushed to version-control.

put all this in a config file and add it in git ignore